.gitignore

```diff
@@ -1,6 +1,7 @@
 .project
 .settings
 node_modules/
+modules/fuse-box-loader-api/
```

Comment:
Well, it is not `ignored` per-se, it is not tracked by git.
For example, it is in `.gitignore`, not `.npmignore`
Everytime you `gulp dist`, it is re-generated anyway : with several branches, it just creates conflicts